actors.md

```diff
@@ -57,16 +57,10 @@ Return: None
 
 ```go
 func SlashConsensusFault(block1, block2 BlockHeader) {
-	if block1.Height != block2.Height {
```

Comment:
Attempting to avoid duplication across files as fault.md states these things.  If pseudocode is preferable maybe we can move it over there?

Reply:
I think the duplication here is okay. I would leave the pseudocode here.

Reply:
Ok adding it in.  It might get a little confusing to write out new slashing rules but will give it a shot.

Reply:
See issue #176 for some thoughts about needed modifications to the spec to make these functions computable from within `SlashConsensusFault`.